src/value/ValueAnnotatedTypeFactory.java

```diff
@@ -877,4 +887,15 @@ private AnnotationMirror convertToUnknown(AnnotationMirror anno) {
         }
         return anno;
     }
+
+    /**
+     * Returns true if {@code anno} is an {@link IntRange} or {@link UnknownVal}.
+     *
+     * @param anno annotation mirror
+     * @return true if {@code anno} is an {@link IntRange} or {@link UnknownVal}
+     */
+    public boolean isIntRange(AnnotationMirror anno) {
```

Comment:
This method seems to be used only once and then only uses the `IntRange` case. Is this really saving any work?